### PR TITLE
Fix getting webapi_token

### PR DESCRIPTION
**Pull request by candela97**
https://github.com/tfedor/AugmentedSteam/pull/1155#issuecomment-848106745
Changed to get webapi tokens from a faster and (hopefully) more reliable api endpoint.

### CODE REVIEW

File: src/js/Content/Features/Community/Inventory/FSetBackgroundOption.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        // Make sure the selected item is a background
+        const viewFullBtn = itemActions.querySelector("a");
+        const m = viewFullBtn && viewFullBtn.href.match(/images\/(items\/\d+\/[a-z0-9.]+)/i);
+        const bgUrl = m && m[1];
+        if (!bgUrl) { return; }
+
+        // Get owned backgrounds and the communityitemid for equipped background
+        if (!this.profileBgsOwned) {
+
+            const response = await RequestData.getJson("https://store.steampowered.com/pointssummary/ajaxgetasyncconfig");
```
Comment (Revadike) [inline src/js/Content/Features/Community/Inventory/FSetBackgroundOption.js:26]:
```suggestion
            // https://github.com/Revadike/UnofficialSteamWebAPI/wiki/Get-Points-Summary-Config
            const response = await RequestData.getJson("https://store.steampowered.com/pointssummary/ajaxgetasyncconfig");
```

File: src/js/Content/Modules/User.js [hunk 2]
```diff
@@ -110,17 +110,6 @@ class User {
...
-    /*
-     * Retrieve access token from user's profile to use in new style WebAPI calls (Services)
-     * Can also be retrieved from the discussions page for apps and the points shop
-     */
-    static async getUserToken() {
-        const html = await RequestData.getHttp(User.profileUrl);
-        const dummyPage = HTMLParser.htmlToDOM(html);
-        const config = dummyPage.getElementById("application_config").dataset.loyaltystore;
-        return JSON.parse(config).webapi_token;
-    }
```
Comment (Revadike) [inline src/js/Content/Modules/User.js:123]: Why not leave it as a separate function? That way, other features could make use of it.

**Comment by Revadike** [comment]
https://github.com/tfedor/AugmentedSteam/pull/1200 is how I would do it

**Comment by MxtOUT** [comment]
What happened to the other changes (FInventoryMarketHelper splitup)

**Comment by candela97** [comment]
In a separate PR #1117. This was based on that, hence the draft status. Sorry for the confusion!

**Pull request closed without merging**